pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -38,6 +38,7 @@ use crate::{IMAGE_FILE_MAGIC, STORAGE_FORMAT_VERSION, TEMP_FILE_SUFFIX};
 use anyhow::{bail, ensure, Context, Result};
 use bytes::Bytes;
 use hex;
+use once_cell::sync::OnceCell;
```

Comment:
was there a problem doing the same transform to DeltaLayer (currently uses OnceCell)?

Reply:
Nevermind :D I was confused. So so ok, we use OnceLock where possible and OnceCell where we need fallible init and it seems that is still waiting on decisions around `Try` on std.